<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSetup</class>
 <widget class="QWidget" name="SerialSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portCombo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Baud Rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudCombo">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataCombo">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stop Bits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopCombo">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parityCombo">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Flow Ctrl</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="flowCombo">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SW</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HW</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Local Echo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="echoCheckBox">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>SerialSetup</receiver>
   <slot>refreshButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>serialPortChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baudCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>baudRateChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>dataChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>stopBitsChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parityCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>parityChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flowCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SerialSetup</receiver>
   <slot>flowControlChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>echoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SerialSetup</receiver>
   <slot>localEchoChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>SerialSetup</receiver>
   <slot>connectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshButtonClicked()</slot>
  <slot>serialPortChanged(QString)</slot>
  <slot>baudRateChanged(QString)</slot>
  <slot>dataChanged(QString)</slot>
  <slot>stopBitsChanged(QString)</slot>
  <slot>parityChanged(QString)</slot>
  <slot>flowControlChanged(QString)</slot>
  <slot>localEchoChanged(bool)</slot>
  <slot>connectButtonClicked()</slot>
 </slots>
</ui>
